### PR TITLE
write bayesfactor to json

### PR DESCRIPTION
**Pull request by sahiljhawar**
Solves #254

### CODE REVIEW

File: nmma/em/analysis.py [hunk 1]
```diff
@@ -744,6 +744,8 @@ def analysis(args):
 
     if args.bestfit:
         bestfit_to_write = bestfit_params.copy()
+        bestfit_to_write["ln_bayes_factor"] = round(result.log_bayes_factor, 3)
```
Comment (tylerbarna) [inline nmma/em/analysis.py:747]: might be better to maintain the 'log' convention even if we mean natural log, as the keys for the likelihood are 'log_likelihood' (unless I'm forgetting and the likelihood is log base 10 and bayes factor is natual log)

File: nmma/em/analysis.py [hunk 2]
```diff
@@ -744,6 +744,8 @@ def analysis(args):
...
+        bestfit_to_write["ln_bayes_factor_err"] = round(result.log_evidence_err, 3)
```
Comment (tylerbarna) [inline nmma/em/analysis.py:748]: any particular reason for rounding? We don't round the other parameters stored in the json

Comment (sahiljhawar) [reply nmma/em/analysis.py:748]: it was rounded upto 3 places in Bilby, so just adopting that :)

Comment (tylerbarna) [reply nmma/em/analysis.py:748]: I'd probably just leave it as the raw number, that way it can be rounded as needed

File: nmma/em/analysis.py [hunk 3]
```diff
@@ -744,6 +744,8 @@ def analysis(args):
 
     if args.bestfit:
         bestfit_to_write = bestfit_params.copy()
+        bestfit_to_write["ln_bayes_factor"] = result.log_bayes_factor
```
Comment (tylerbarna) [inline nmma/em/analysis.py:747]:
```suggestion
        bestfit_to_write["log_bayes_factor"] = result.log_bayes_factor
```

File: nmma/em/analysis.py [hunk 4]
```diff
@@ -744,6 +744,8 @@ def analysis(args):
...
+        bestfit_to_write["ln_bayes_factor_err"] = result.log_evidence_err
```
Comment (tylerbarna) [inline nmma/em/analysis.py:748]:
```suggestion
        bestfit_to_write["log_bayes_factor_err"] = result.log_evidence_err
```

**Review by tylerbarna — APPROVED** [review]

**Pull request merged**